sourced/ml/__main__.py

```diff
@@ -66,6 +66,22 @@ def add_default_args(my_parser):
         choices=[ex.NAME for ex in extractors.__extractors__.values()],
         required=True, help="The feature extraction scheme to apply.")
 
+    repos2ids_parser = subparsers.add_parser(
+        "repos2ids", help="Convert source code to a token identifier stream."
```

Comment:
The output should be an ASDF model file with all the unique identifiers.

A stream is pipe or smth non-physical, it is not a file.

Reply:
Also please put a comment here how it is different from repos2df and why we need to invent it